www/docs/client/links/httpBatchLink.md

```diff
@@ -144,6 +144,46 @@ export const trpc = createTRPCNext<AppRouter>({
 });
 ```
 
+## Streaming requester (experimental)
+
+> ⚠️ `streamRequester` is experimental and may change in the future.
+
+When batching requests together, the default behavior is to wait for all requests to finish before sending the response. This standard behavior uses the `batchRequester` under the hood. If you want to send responses as soon as they are ready, you can use the `streamRequester` instead. This is useful for long-running requests.
+
+```ts title="client/index.ts"
+import {
+  createTRPCProxyClient,
+  httpBatchLink,
+  unstable_streamRequester,
+} from '@trpc/client';
+import type { AppRouter } from '../server';
+
+const client = createTRPCProxyClient<AppRouter>({
+  links: [
+    httpBatchLink({
+      url: 'http://localhost:3000',
+      // 👇 enable streaming mode
+      requester: unstable_streamRequester,
+    }),
+  ],
+});
+```
+
+Using the `streamRequester` will:
+
+- Cause the requests to be sent with a `X-Trpc-Batch-Mode: stream` header
```

Comment:
I think we can just do `trpc-xxx` :D 

<img width="659" alt="CleanShot 2023-05-23 at 09 24 07@2x" src="https://github.com/trpc/trpc/assets/459267/85b229da-3354-4007-a54d-1792946560e4">